mock/py/mockbuild/util.py

```diff
@@ -1339,6 +1339,10 @@ def check_config(config_opts):
 
 @traceLog()
 def include(config_file, config_opts, is_statement=False):
+    if not os.path.isabs(config_file):
+        config_dir = os.path.dirname(config_opts.config_file)
+        config_file = find_config_in_path(config_file, config_dir)
```

Comment:
I'd prefer not to split the argument to `config_file` and `config_dir` ... it is still just the same value filtered by `dirname`.  Why not to have `find_config_in_path(config_file)` - and make the method search through predictible set of directories (e.g. given by config option)?

Reply:
It is not the same value, actually. The `config_opts.config_file` contains the value, that you passed via `-r` parameter, e.g. `../mock-core-configs/etc/mock/fedora-rawhide-x86_64.cfg`. While the `config_file` contains the currently included config/template file, e.g. `fedora-rawhide.tpl` (with this PR, otherwise the template has full etc path)

Reply:
Aha, ok, this isn't obvious  - good point.